src/CSSFramework/BootstrapCSSFramework.php

```diff
@@ -64,62 +42,44 @@ public function getColumnClasses()
         return implode(' ', $classes);
     }
 
-    /**
-     * Get title class from base element.
-     *
-     * @return string
-     */
-    public function getTitleSizeClass()
+    public function getTitleSizeClass(): string
     {
-        return $this->baseElement->TitleClass;
+        return $this->baseElement->TitleClass ?? strtolower($this->baseElement->TitleTag);
```

Comment:
This means:

return "$this->baseElement->TitleClass" or if that is null return strtolower($this->baseElement->TitleClass). I don't think that's what you intend to do.

Reply:
Never mind, I misread it!